pkg/urbit/Makefile

```diff
@@ -45,6 +46,23 @@ mrproper: clean
 
 ################################################################################
 
+include/ca-bundle.h:
+	@echo XXD -i $(SSL_CERT_FILE)
+	@cat $(SSL_CERT_FILE) > include/ca-bundle.crt
+	@xxd -i include/ca-bundle.crt > include/ca-bundle.h
+	@rm include/ca-bundle.crt
```

Comment:
It looks like this was inadvertently added. This is the old approach, now handled through nix dependencies and the `./configure` script.